collectors/scm/gitlab/src/main/java/com/capitalone/dashboard/gitlab/GitlabUrlUtility.java

```diff
@@ -73,17 +87,155 @@ public URI buildApiUrl(GitlabGitRepo repo, boolean firstRun, int resultsPerPage)
 				.queryParam(PER_PAGE_QUERY_PARAM_KEY, resultsPerPage)
 				.build(true).toUri();
 
-		LOG.info("---> Gitlab URI: " + uri.toString());
+		LOG.info("---> Gitlab commits URI: " + uri.toString());
 		return uri;
     }
 
-    private String getProtocol() {
-        return StringUtils.isBlank(gitlabSettings.getProtocol()) ? DEFAULT_PROTOCOL : gitlabSettings.getProtocol();
-    }
+	public URI buildIssuesApiUrl(GitlabGitRepo repo, boolean firstRun, int resultsPerPage) {
+		String repoUrl = repo.getRepoUrl();
+        if (repoUrl.endsWith(GIT_EXTENSION)) {
```

Comment:
Formatting is off here.

Reply:
Changed to space!

https://github.com/Hygieia/Hygieia/pull/2755/commits/3cbb00eff8b1daf70388ea69d7c1d03eb0682045#diff-a1c5a6b5d72aaed0621398f4d715f22bR94